### ISSUE TITLE
Comments get removed from formated Svelte Component

### ISSUE THREAD

**Issue by SaintPepsi** [sveltejs/prettier-plugin-svelte#207]
Hi when I format a svelte component, any comment that is at the end of an element gets removed.

Gets removed:
```
<div>
    <div/>
    <!-- testing 123 -->
</div>
```

Doesn't get removed:
```
<div>
    <!-- testing 123 -->
    <div/>
</div>
```

Any solutions?

**Comment by dummdidumm** [comment]
I cannot reproduce this given your code snippet. Which version of the plugin are you using?

**Comment by SaintPepsi** [comment]
Hi, I use Svelte for VS Code v 104.4.1.

**Comment by SaintPepsi** [comment]
I just discovered something else.

It only happens when you have style tags with styling in them.

Following removes last comment on format
```
<div>
  <!-- test -->
  <p />
  <!-- test -->
</div>
<style>
</style>
```

**Comment by SaintPepsi** [comment]
Thanks for fixing that so quickly!

Kind Regards,
Ian Hogers

Email Signature, phone: 0458 452 419, email: info@ianhogers.com

**Comment by dummdidumm** [comment]
I'll write in here once it's released for VS Code

**Comment by dummdidumm** [comment]
This was released in the VS Code extension version 104.4.2

**Comment by SaintPepsi** [comment]
That's great! Thanks.
